### DISCUSSION TITLE
Jobs Board for vscode extension developers

### DISCUSSION THREAD

**Discussion by ssbarnea** [microsoft/vscode-discussions#48] [Extension Show and Tell]
As that would likely be one of the best places to meet others that have experience developing vscode extensions, it would make sense to have *just* one thread for advertising jobs that would involve this.

I would advise not to spam the discussion board with one thread per posting and reuse this one. Anyone interested will be able to subscribe to the topic.

**Comment by ssbarnea** [comment]
Ansible DevTools team at [Red Hat](https://www.redhat.com) is adding a [new engineer position in India](https://global-redhat.icims.com/jobs/95745/principal-software-engineer/job) so **Python and TypeScript** are highly valued. That listing advertise it as principal, but senior will also do. One product that we work on is [Ansible vscode extension](https://marketplace.visualstudio.com/items?itemName=redhat.ansible) and  all our team doings are **open-source** and public.

**Comment by progrium** [comment]
Bump. Also looking for extension developers to work with on contract basis and can be from anywhere. Working on all kinds of fun stuff and it's all open source, but it's just me right now. 

Also would love to know any other place to find them!
